pipeline/plugin.go

```diff
@@ -33,6 +33,18 @@ type OutputPlugin interface {
 	Out(*Event)
 }
 
+type PluginsStarterData struct {
+	Config AnyConfig
+	Params *OutputPluginParams
+}
+type PluginsStarterMap map[string]PluginsStarterData
+
+type OutputPlugins interface {
```

Comment:
I don't find usage, is it necessary ?

Reply:
Correct, I'll add interface dep in s3 plugin or drop this iface.